UIComponents/Sources/UIComponents/Onboarding/OnboardingStepView.swift

```diff
@@ -38,28 +38,28 @@ public final class OnboardingImageStepView: UIView {
         return item
     }()
 
-    private let imageView: UIImageView = {
+    private lazy var imageView: UIImageView = {
```

Comment:
Could you please clarify this change?

Reply:
Due to the use of the designEngine property at the time this property is created

Reply:
Strange, I don't see `designEngine` property usage here